### ISSUE TITLE
Create more meaningful opensearch index names

### ISSUE THREAD

**Issue by arisjr** [google/timesketch#2596]
**Is your feature request related to a problem? Please describe.**
When a user needs to find a problematic openseach index from a problematic sketch, he need to struggle to find the right one if there's various sketches and the system is accessed by multiple users.

**Describe the solution you'd like**
Insert the SKETCH-ID as a prefix on the index name. (Example: 2-600a04c642e146e9ae68a5513bf8add7)

**Describe alternatives you've considered**
Insert also the sketch name somehow.

**Comment by jaegeral** [comment]
Yeah that request makes sense, I am moving it to Q3 where we might have time to look in feature requests

**Comment by berggren** [comment]
The naming scheme is from back when TS supported an index to be part of multiple sketches. This functionality has since been deprecated.

@arisjr  What is your use-case in this case? Do you need to manually go into opensearch and operate on the index?

Would it be sufficient to have a more visible list of indexes (with timeline mapping, i.e. what timelines are using an index)?

**Comment by arisjr** [comment]
Hello, @berggren.

I had, in the past, made some manual maintenance on opensearch indexes and felt this difficulty to isolate the right index - it's documented on another issue on here (https://github.com/google/timesketch/issues/2353).

Also, in the past, for one of our analysts, I had to enable opensearch-dashboard on one index made by timesketch. Also, the index identification was important in this case. 

So, I thought the more meaningful index names would be of great help on this scenarios.
